x/stakeibc/keeper/update_redemption_rates_test.go

```diff
@@ -43,96 +45,97 @@ func (s *KeeperTestSuite) SetupUpdateRedemptionRates(
 	toBeTransferedDepositRecord := recordtypes.DepositRecord{
 		HostZoneId: "GAIA",
-		Amount:     int64(JUST_DEPOSITED_BAL),
+		Amount:     int64(justDepositedBal),
 		Status:     recordtypes.DepositRecord_TRANSFER,
 	}
 	s.App.RecordsKeeper.AppendDepositRecord(s.Ctx, toBeTransferedDepositRecord)
 
 	// set the stSupply by minting to a random user account
 	user := Account{
 		acc:           s.TestAccs[0],
-		stAtomBalance: sdk.NewInt64Coin(stAtom, int64(ST_SUPPLY)),
+		stAtomBalance: sdk.NewInt64Coin(stAtom, int64(stSupply)),
 	}
 	s.FundAccount(user.acc, user.stAtomBalance)
 
 	// set the staked balance on the host zone
 	hostZone := stakeibc.HostZone{
 		ChainId:        "GAIA",
 		HostDenom:      "uatom",
-		StakedBal:      STAKED_BAL,
-		RedemptionRate: REDEMPTION_RATE_T0,
+		StakedBal:      stakedBal,
+		RedemptionRate: redemptionRateT0,
 	}
 	s.App.StakeibcKeeper.SetHostZone(s.Ctx, hostZone)
 
 	return UpdateRedemptionRatesTestCase{
-		host_zone:          hostZone,
-		redemption_rate_t0: REDEMPTION_RATE_T0,
-		staked_bal:         STAKED_BAL,
-		undelegated_bal:    UNDELEGATED_BAL,
-		just_deposited_bal: JUST_DEPOSITED_BAL,
-		st_supply:          ST_SUPPLY,
+		hostZone:         hostZone,
+		redemptionRateT0: redemptionRateT0,
+		stakedBal:        stakedBal,
+		undelegatedBal:   undelegatedBal,
+		justDepositedBal: justDepositedBal,
+		stSupply:         stSupply,
+		allRecords:       []recordtypes.DepositRecord{toBeStakedDepositRecord, toBeTransferedDepositRecord},
 	}
 }
 
 func (s *KeeperTestSuite) TestUpdateRedemptionRatesSuccessful() {
 
-	STAKED_BAL := uint64(5)
-	UNDELEGATED_BAL := uint64(3)
-	JUST_DEPOSITED_BAL := uint64(3)
-	ST_SUPPLY := uint64(10)
+	stakedBal := uint64(5)
+	undelegatedBal := uint64(3)
+	justDepositedBal := uint64(3)
+	stSupply := uint64(10)
 
-	REDEMPTION_RATE_T0 := sdk.NewDec(1)
-	tc := s.SetupUpdateRedemptionRates(STAKED_BAL, UNDELEGATED_BAL, JUST_DEPOSITED_BAL, ST_SUPPLY, REDEMPTION_RATE_T0)
+	redemptionRateT0 := sdk.NewDec(1)
+	tc := s.SetupUpdateRedemptionRates(stakedBal, undelegatedBal, justDepositedBal, stSupply, redemptionRateT0)
 
 	// sanity check on inputs (check redemptionRate at genesis is 1)
-	s.Require().Equal(tc.redemption_rate_t0, sdk.NewDec(1))
+	s.Require().Equal(tc.redemptionRateT0, sdk.NewDec(1))
 
-	records := s.App.RecordsKeeper.GetAllDepositRecord(s.Ctx)
+	records := tc.allRecords
 	s.App.StakeibcKeeper.UpdateRedemptionRates(s.Ctx, records)
 
-	hz, found := s.App.StakeibcKeeper.GetHostZone(s.Ctx, tc.host_zone.ChainId)
+	hz, found := s.App.StakeibcKeeper.GetHostZone(s.Ctx, tc.hostZone.ChainId)
 	s.Require().True(found)
 	rrNew := hz.RedemptionRate
 
-	s.Require().Equal(rrNew, sdk.NewDec(int64(STAKED_BAL)+int64(UNDELEGATED_BAL)+int64(JUST_DEPOSITED_BAL)).Quo(sdk.NewDec(int64(ST_SUPPLY))))
+	expectedNewRate := sdk.NewDec(5 + 3 + 3).Quo(sdk.NewDec(10))
+	s.Require().Equal(rrNew, expectedNewRate)
 	s.Require().Equal(rrNew, sdk.NewDec(11).Quo(sdk.NewDec(10)))
```

Comment:
nit: no need to keep both of these, you can remove the second

Reply:
done